src/python_bring_api/bring.py

```diff
@@ -87,6 +89,8 @@ def login(self) -> BringAuthResponse:
             raise BringAuthException('Login failed due to missing data in the API response, please check your email and password.')
         
         self.uuid = data['uuid']
+        self.publicUuid = data.get('publicUuid', "")
+        self.bringListUUID = data.get('bringListUUID', "")

```

Comment:
What does this uuid do?

Reply:
publicUuid is similar to the users uuid and is required for sending notifications.
bringListUUID is the users default shopping list, i thought it might come in handy, you can ommit calling loadLists and just use bring.ListUUID when calling saveItem e.g.

Reply:
Maybe add a comment to the readme for that

Reply:
i thought of using it like this


```
bring.saveItem(
    bring.bringListUUID,
    "Koriander"
)
```
but you are right, was also thinking of rewriting all methods so that they can be called without the ListUuid Parameter but that should be a separate PR

Reply:
I would be careful for when rewriting to make sure, current implementations do not break.